app/src/main/java/com/muzima/model/shr/kenyaemr/Addendum/Addendum.java

```diff
@@ -0,0 +1,38 @@
+package com.muzima.model.shr.kenyaemr.Addendum;
+
+/**
+ * Created by patrick on 11/04/2018.
+ */
+import com.fasterxml.jackson.annotation.*;
+import com.muzima.model.shr.kenyaemr.CardDetails;
+
+import java.util.List;
+
+/**
+ * Created by Muhoro on 2/27/2018.
+ */
+
+public class Addendum {
+
+    @JsonProperty("CARD_DETAILS")
+    private CardDetails CardDetail;
```

Comment:
Variable naming is against the standards. Should be cardDetails i.e lower case c